packages/dara-core/dara/core/main.py

```diff
@@ -423,3 +429,40 @@ def start(extra=None):
     config._run_discovery(config_module)
 
     return _start_application(config=config._to_configuration())
+
+
+class CustomResponse(Response):
```

Comment:
Can we move it somewhere within `dara.core.internals` to not pollute the main file?